app/src/main/java/com/internship/move/ui/splash/SplashViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.internship.move.ui.splash
+
+import androidx.lifecycle.ViewModel
+import com.internship.move.repository.Repository
+
+class SplashViewModel(
+    private val repo: Repository
+) : ViewModel() {
+
+    fun getIsLoggedIn() = repo.getIsLoggedIn()
```

Comment:
Just a syntax sugar suggestion: Making these methods into variables with custom getters would make the calling code look cleaner and easier to read